### PR TITLE
release: fixes

### PR DESCRIPTION
**Pull request by XVincentX**
1. Remove packages that do not make sense
2. Make sure pkg has all the data it needs to build a working binary
3. Do not recreate the package.json with weird stuff
4. Use `npm version from-git` so we just need a tag to release
5. Create a Docker image (this is significantly easier than Prism)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,5 @@
 *.log*
 *-debug.log*
 *-error.log*
+oclif.manifest.json
+binaries
```
Comment (P0lip) [inline .gitignore]:
```suggestion
/binaries
```

**Comment by XVincentX** [comment]
@P0lip I wish — Prism uses kind of a different schema since it's a monorepo and follows different rules  . I've basically re-applied the knowledge I gained while automatic Express Gateway.

File: .circleci/config.yml [hunk 2]
```diff
@@ -102,9 +103,13 @@ workflows:
       - release:
           filters:
             branches:
-              only: master
+              ignore: /.*/
+            tags:
+              only: /^v.*/
```
Comment (P0lip) [inline .circleci/config.yml]:
```suggestion
              only: /^v\.*/
```
Pretty sure you want to match a dot here.

File: .circleci/config.yml [hunk 3]
```diff
@@ -102,9 +103,13 @@ workflows:
...
       - upload_artifacts:
           filters:
+            branches:
+              ignore: /.*/
             tags:
               only: /^v.*/
```
Comment (P0lip) [inline .circleci/config.yml]: Ditto?

**Pull request merged**